region/crateria/east.json

```diff
@@ -1482,6 +1482,24 @@
                   ],
                   "devNote": "Could use a failure definition?"
                 },
+                {
+                  "name": "West Ocean Bug Ice Clip",
+                  "notable": true,
+                  "requires": [
+                    "canNonTrivialIceClip"
+                  ],
+                  "note": [
+                    "Freeze a bug a three tiles to the right of the morph tunnel entrance, directly under the tile where ceiling is higher.",
+                    "Spin jump onto the bug, crouch, and jump up.",
+                    "With HiJump it is more lenient, otherwise the bug must be at an exact pixel height.",
+                    "If you have XRay and Morph, it can be done more easily:",
```

Comment:
This is normal for `canNonTrivialIceClip` unless you are introducing the bug height needed for xray clip with this.

As an aside, Morph, XRay, and canPartialFloorClip can traverse this link with these items.

Reply:
True, that seems to take away most of the reason to detail how to use Morph & XRay in the ice clip. I simplified the note now by cutting that part out.